plugins/adrv9002.c

```diff
@@ -2281,11 +2281,24 @@ static bool profile_gen_check_api(gpointer data)
 		goto err;
 	}
 
-	if(strcmp(version, supported_version) != 0) {
-		sprintf(message, "\nOnly API version %s is supported, the device uses %s!", supported_version, version);
-		profile_gen_set_debug_info(data, message);
-		free(supported_version);
-		goto err;
+	if(strstr(version, supported_version) != NULL && strstr(supported_version, version) != NULL) {
```

Comment:
You are making this condition too permissive:
For example this will pass:
```
version = "v68.8.1"
supported_version = "8.8.1"
```
Also this will pass:
```
version = "v68.8.1"
supported_version = "8.1"
```
Also this will pass:
```
version = "v68.8.1"
supported_version = "1"
```
It would make sense to consider the minimum valid string for a version (for example: "68.8.1") and find that substring in strings like: "v68.8.1", "ver68.8.1", etc.